corehq/apps/auditcare/tests/test_export.py

```diff
@@ -250,6 +252,51 @@ def test_related_query_error(self):
     def test_get_sql_start_date(self):
         self.assertEqual(get_sql_start_date(), datetime(2021, 2, 1, 3))
 
+    def test_get_domain_first_access_times(self):
+
+        def create_session_events(domain):
+            login_event = dict(
+                user=self.username,
+                domain=domain,
+                session_key=uuid.uuid4().hex,
+                event_date=datetime.utcnow(),
+            )
+            for domain, minutes in [(None, -1),
```

Comment:
Reusing `domain` here is confusing since it's already defined and referenced above. Consider renaming.